### PR TITLE
Adds Firebird

### PR DESCRIPTION
**Pull request by TemperanceTempy**
BIRD UP
Adds WAW abnormality Firebird and its associated EGO (Minus the weapon cause that was already in the game)

## About The Pull Request

BIRD UP
Adds WAW abnormality Firebird and its associated EGO (Minus the weapon cause that was already in the game)

## Why It's Good For The Game

New Abno

## Changelog
:cl:
add: Firebird
soundadd: Special sound for bird dashing
imageadd: Feather of Honor EGO Gift
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+		can_act = FALSE
+		var/dir_to_target = get_dir(src, target)
+		var/turf/T = get_turf(src)
+		for(var/i = 1 to dash_max)
+			T = get_step(T, dir_to_target)
+			if(T.density)
+				if(i < 4) // Mob attempted to dash into a wall too close, stop it
+					can_act = TRUE
+					return
+				break
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm:125]: Why the break this isn't a loop?

File: code/datums/abnormality/_ego_datum/waw.dm [hunk 2]
```diff
@@ -191,3 +191,8 @@
 /datum/ego_datum/weapon/wings
 	item_path = /obj/item/ego_weapon/wings
 	cost = 50
+
+// Fire Bird - Feather of Honor
```
Comment (Kitsunemitsu) [inline code/datums/abnormality/_ego_datum/waw.dm:195]: Weapon Datum doesn't actually exist.

File: code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+	faction = list("hostile")
+	can_breach = TRUE
+	start_qliphoth = 3
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.4, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 2.0)
+	light_color = COLOR_LIGHT_ORANGE
+	light_range = 0
+	light_power = 0
+	loot = list(/obj/item/gun/ego_gun/feather)
+	ego_list = list(/datum/ego_datum/armor/feather)
+	gift_type = /datum/ego_gifts/feather
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm:35]: Change these variables to something more fitting. Right now it nuzzles you.
While cute, it's not actually fitting
friendly_verb_continuous = ""
friendly_verb_simple = ""

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Waiting on status sprites; worksed mostly as intended.

File: code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+	base_pixel_y = -16
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(55, 55, 50, 50, 60),
+		ABNORMALITY_WORK_INSIGHT = list(30, 30, 25, 25, 35),
+		ABNORMALITY_WORK_ATTACHMENT = list(45, 45, 40, 40, 50),
+		ABNORMALITY_WORK_REPRESSION = list(45, 45, 40, 40, 50)
+		)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+	faction = list("hostile")
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm:26]: If you want firebird to ignore players and not follow them you can use
list("hostile", "neutral")

**Comment by Kitsunemitsu** [comment]
Also, we do not merge Abno PRs without records.

File: code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm (hunk 3 above)
Comment (TemperanceTempy) [reply code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm:35]: I gave the the verb "grazes" for now, as itd probably pass by people, does that work?

**Comment by TemperanceTempy** [comment]
> Also, we do not merge Abno PRs without records.

Ah, do I add the records myself? or how would i go about getting the records set up?

**Review by Kitsunemitsu — APPROVED** [review]
Looks good to me

**Comment by Kitsunemitsu** [comment]
Fuck I forgot to squash I didn't see the 2nd commit 💀 

This is why we have squash and merge.

**Pull request merged**